### ISSUE TITLE
Standardize new user onboarding around the "monitoring event pipeline" workflow

### ISSUE THREAD

**Issue by calebhailey** [sensu/sandbox#3]
TL;DR: I propose we establish this repository as **the source** for learning Sensu going forward. I've asked @jspaleta to be the lead maintainer for the Sensu Core experience, and @mattbeninghof to lead on the Sensu Enterprise experience.

## The longer version
This repository was created at an interesting time in the history of the Sensu project and company. At Sensu Inc we had been raising awareness internally around developing better tools for helping new users to learn a) what Sensu is, and b) how to get started using it. This internal effort has resulted in a number of new projects, from internal "dogfooding" guides, to sales-focused workshops for new Sensu Enterprise users. All of these are the right idea! However, in order to help make this easier for Sensu users, it would be ideal if we could rally around a consistent story and single path. 

Fortunately, around the same time as this repository was created, the company has arrived on what it believes may become a compelling new narrative and messaging for new and experienced Sensu users alike. That messaging is describing Sensu as a "monitoring event pipeline", and helping users understand how to leverage this monitoring event pipeline to deliver a true "workflow automation for monitoring" solution. If early user feedback is any indication, this new framing will have a dramatic affect on simplifying the onboarding experience for new users. 

For this reason, we would like to kickoff what we hope to be a long-running resource for the Sensu community by making this sensu/sandbox repo _the_ canonical starting point for new users to learn Sensu. And the consistent messaging we want to use to introduce Sensu to those users should be the monitoring event pipeline.

As mentioned above, our developer advocacy team (@jspaleta) and sales engineering team (@mattbeninghof) will head up efforts for Sensu Core and Sensu Enterprise workflows, respectively. Looking forward to following the progress here!

**Comment by apaskulin** [comment]
We successfully standardized on the monitoring event pipeline workflow using the sandbox projects in this repo :Tada:
